package.json

```diff
@@ -185,8 +185,14 @@
         "category": "ZenML Stacks"
       },
       {
-        "command": "zenml.renameStack",
-        "title": "Rename Stack",
+        "command": "zenml.createStack",
+        "title": "Create Stack",
```

Comment:
register... etc (to be changed everywhere for both stacks and components

Reply:
Have changed all instances of create to register; code, errors, and comments.